src/web/src/alert_list.ecpp

```diff
@@ -153,10 +153,9 @@ UserInfo user;
 
     {
         int64_t dbid =  persist::name_to_asset_id (checked_asset);
-        if (dbid == -1) {
+        if (!checked_asset.empty () && dbid == -1) {
             http_die ("element-not-found", checked_asset.c_str ());
         }
-
         element_id = (uint32_t) dbid;
```

Comment:
Just in case: is it acceptable for us to continue with `dbid == -1` (and so casted to `element_id = (uint32_t)(2^32-1)` when asset IS empty)?